docker.go

```diff
@@ -106,24 +106,32 @@ func (d *dockerContainerService) syncContainers(now time.Time) {
 			continue
 		}
 
-		containerIP := container.NetworkSettings.IPAddress
+		var containerIP []string
+		if container.NetworkSettings.IPAddress != "" {
+			containerIP = append(containerIP, container.NetworkSettings.IPAddress)
+		}
+		for network := range container.NetworkSettings.Networks {
+			containerIP = append(containerIP, container.NetworkSettings.Networks[network].IPAddress)
+		}
```

Comment:
I should have done this before, but add a check for an empty `containerIPs` list. Log an error and continue if there are no IPs. I don't know if it's possible to not have any IPs, but it would be tough to debug.